sdk/python/v1beta1/kubeflow/katib/constants/constants.py

```diff
@@ -56,3 +57,6 @@
 BASE_IMAGE_MXNET = "docker.io/mxnet/python:1.9.1_native_py3"
 
 DEFAULT_DB_MANAGER_ADDRESS = "katib-db-manager.kubeflow:6789"
+
+# Test result constants
+TEST_RESULT_SUCCESS = "success"
```

Comment:
Please keep this variable in the `katib_client_test.py` since we use it only in that file.